src/current/v24.2/install-cockroachdb-windows.md

```diff
@@ -53,40 +62,44 @@ Use one of the options below to install CockroachDB. To upgrade an existing clus
+{{site.data.alerts.end}}
 
-For CockroachDB v22.2.beta-5 and above, Docker images are <a href="https://docs.docker.com/build/building/multi-platform/">multi-platform images</a> that contains binaries for both Intel and ARM. CockroachDB on ARM systems is <b>experimental</b> and is not yet qualified for production use and not eligible for support or uptime SLA commitments. Multi-platform images do not take up additional space on your Docker host.
+For CockroachDB v22.2 and above, Docker images are [multi-platform images](https://docs.docker.com/build/building/multi-platform/) that contains binaries for both Intel and ARM. Multi-platform images do not take up additional space on your Docker host.
 
-Docker images for previous releases contain Intel binaries only. Intel binaries can run on ARM systems, but with a significant reduction in performance.
+Intel binaries can run on ARM systems, but with a significant reduction in performance.
 
 1. Install <a href="https://docs.docker.com/docker-for-windows/install/">Docker for Windows</a>.
 
     {{site.data.alerts.callout_success}}
     Docker for Windows requires 64bit Windows 10 Pro and Microsoft Hyper-V. Please see the <a href="https://docs.docker.com/docker-for-windows/install/#what-to-know-before-you-install">official documentation</a> for more details. Note that if your system does not satisfy the stated requirements, you can try using <a href="https://docs.docker.com/toolbox/overview/">Docker Toolbox</a>.
     {{site.data.alerts.end}}
 
-1. In PowerShell, confirm that the Docker daemon is running in the background:
+1. In PowerShell, confirm that Docker is running in the background:
 
     {% include_cached copy-clipboard.html %}
     ~~~ powershell
     docker version
     ~~~
 
-    If you see an error, start Docker for Windows.
+    If you see an error, verify your Docker for Windows installation, then try again.
 
-1. <a href="https://docs.docker.com/docker-for-windows/#/shared-drives">Share your local drives</a>. This makes it possible to mount local directories as data volumes to persist node data after containers are stopped or deleted.
+1. [Share your local drives](https://docs.docker.com/docker-for-windows/#/shared-drives) so you can mount local directories as data volumes to persist node data after containers are stopped or deleted.
 
-1. In PowerShell, pull the image for the {{page.release_info.version}} release of CockroachDB from <a href="https://hub.docker.com/r/{{page.release_info.docker_image}}/" class="win-docker-step3" id="win-docker-step3-{{page.version.version}}" data-eventcategory="win-docker-step3">Docker Hub</a>:
+1. Pull the image for CockroachDB from [Docker Hub](https://hub.docker.com/r/{{page.release_info.docker_image}}/). <a id="win-docker-step3-{{ page.version.version }}"></a>:
```

Comment:
This link is rendering as `https://hub.docker.com/r/cockroachdb/cockroach/`, so perhaps the variable isn't working as intended?

```suggestion
1. <a id="win-docker-step3-{{ page.version.version }}"></a> Pull the image for CockroachDB from [Docker Hub](https://hub.docker.com/r/{{page.release_info.docker_image}}/):
```